doc/source/tutorial/optimize.rst

```diff
@@ -1557,12 +1560,26 @@ We have a table showing the individual medley times of five students:
  E          46.3           47.8         50.4        37.2
 ==========  ===========  ============  ===========  ===============================
 
-We need to choose best four students for each swimming style.
-This is a typical linear sum assignment problem, we can use :func:`linear_sum_assignment` to solve it.
+We need to choose the best student for each of the four swimming styles.
+This is a typical linear sum assignment problem. We can use :func:`linear_sum_assignment` to solve it.
+
+The linear sum assignment problem is one of the most famous combinatorial optimization problems.
+When we are given a matrix :math:`C` known as a "cost matrix" and we want to match each row to a different column,
+we need to find a combinations of a row and a column which the sum of the corresponding entries is minimized
+(or maximized).
+
+Formally, let X be a boolean matrix where :math:`X[i,j] = 1` iff row i is assigned to column j.
+Then the optimal assignment has cost
+
+.. math::
+
+    \min \sum_i \sum_j C_{i,j} X_{i,j}
 
 First of all, we need to setup a cost matrix.
 :func:`linear_sum_assignment` can assign each row to the best column.
-We can setup the cost matrix based on the upper table:
+In this example, we want to assign each swimming style to a student.
+This means that rows of the cost matrix show each swimming style and columns show each student.
+In other words, the upper table needs to be transposed for the cost matrix:
```

Comment:
```suggestion
Therefore, the table above needs to be transposed so that the rows correspond
with swimming styles and the columns correspond with students:
```